testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):

```

Comment:
I think we should also test the case in which init_order and final_order are of different length - how our function handle that case? I believe it should throw an error of the sort "These are not the same car parks!" or "There is a mismatching in the car parks' sizes, please check them again". Something like that.